File: Service/CustomModels/Admin/Branch/BranchListInit.cs
namespace Service.CustomModels.Admin.Branch;

public class BranchListInit
{
    public int BranchId { get; set; }

    public int CompanyId { get; set; }

    public string BranchName { get; set; } = null!;

    public string BranchCode { get; set; } = null!;

    public string? Address { get; set; }

    public string? ManagerName { get; set; }

    public int? Phone { get; set; }

    public int Status { get; set; }
    
    public string StatusName { get; set; }

    public int CreatedBy { get; set; }
    
    public string CreatedByName { get; set; }

    public DateTime CreatedDate { get; set; }

    public int? ModifiedBy { get; set; }
    
    public string ModifiedByName { get; set; }

    public DateTime? ModifiedDate { get; set; }
}

File: Studiopro/src/app/models/admin/branch/BranchListInit.ts
interface BranchListInit {
     BranchId: number;

    CompanyId: number;

    BranchName: string;

    BranchCode: string;

    Address: string | null;

    ManagerName: string | null;

    Phone: number | null;

    Status: number;
    
    StatusName: string;

    CreatedBy: number;
    
    CreatedByName: string;

    CreatedDate: Date | string;

    ModifiedBy: number | null;
    
    ModifiedByName: string;

    ModifiedDate: Date | string | null;
}